src/Raven.Server/Documents/Queries/GraphQueryRunner.cs

```diff
@@ -145,8 +147,9 @@ public async Task WriteDetailedQueryResult(IndexQueryServerSide indexQuery, Quer
                         idc = new IncludeDocumentsCommand(Database.DocumentsStorage, queryContext.Documents, query.Metadata.Includes, fieldsToFetch.IsProjection);
                         icevc = IncludeCompareExchangeValuesCommand.ExternalScope(queryContext, query.Metadata.CompareExchangeValueIncludes);
                         irc = new IncludeRevisionsCommand(database: Database, context: queryContext.Documents, query.Metadata.RevisionIncludes);
+                        itsc = new IncludeTimeSeriesCommand(context: queryContext.Documents, query.Metadata.TimeSeriesIncludes?.TimeSeries ?? null);
+                        icc = new IncludeCountersCommand(database: Database, context: queryContext.Documents, query.Metadata.CounterIncludes?.Counters ?? null);
```

Comment:
I think we can leave those as nulls

Reply:
That was my initial thought and how I implemented it, but then it provided null values and made null metadata, and it throws.
I made it that way so we could take care of it in `IncludeCountersCommand`.